src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -474,16 +473,17 @@ namespace DataHeatBalance {
         bool HasLtsRetAirGain;       // TRUE means that zone lights return air heat > 0.0 calculated from plenum temperature
         bool HasAirFlowWindowReturn; // TRUE means that zone has return air flow from windows
         // from refrigeration cases for this zone
-        Real64 InternalHeatGains;                     // internal loads (W)
-        Real64 NominalInfilVent;                      // internal infiltration/ventilation
-        Real64 NominalMixing;                         // internal mixing/cross mixing
-        bool TempOutOfBoundsReported;                 // if any temp out of bounds errors, first will show zone details.
-        bool EnforcedReciprocity;                     // if zone/space required forced reciprocity -- less out of bounds temp errors allowed
-        int ZoneMinCO2SchedIndex;                     // Index for the schedule the schedule which determines minimum CO2 concentration
-        int ZoneMaxCO2SchedIndex;                     // Index for the schedule the schedule which determines maximum CO2 concentration
-        int ZoneContamControllerSchedIndex;           // Index for this schedule
-        bool FlagCustomizedZoneCap;                   // True if customized Zone Capacitance Multiplier is used
-        std::vector<std::string> otherEquipFuelTypes; // List of fuel types used by other equipment in this zone
+        Real64 InternalHeatGains;           // internal loads (W)
+        Real64 NominalInfilVent;            // internal infiltration/ventilation
+        Real64 NominalMixing;               // internal mixing/cross mixing
+        bool TempOutOfBoundsReported;       // if any temp out of bounds errors, first will show zone details.
+        bool EnforcedReciprocity;           // if zone/space required forced reciprocity -- less out of bounds temp errors allowed
+        int ZoneMinCO2SchedIndex;           // Index for the schedule the schedule which determines minimum CO2 concentration
+        int ZoneMaxCO2SchedIndex;           // Index for the schedule the schedule which determines maximum CO2 concentration
+        int ZoneContamControllerSchedIndex; // Index for this schedule
+        bool FlagCustomizedZoneCap;         // True if customized Zone Capacitance Multiplier is used
+        std::vector<ExteriorEnergyUse::ExteriorFuelUsage> otherEquipFuelTypeNums; // List of fuel types used by other equipment in this zone
+        std::vector<std::string> otherEquipFuelTypeNames;                         // List of fuel types used by other equipment in this zone
```

Comment:
Do we even need a vector of fuel type names if we have a list of fuel type nums? I am not saying to change anything here, just thinking about the future.

Reply:
@rraustad you beat me to this comment! Love it! In general, there is no need to store the name of an enumerated type. That can always be retrieved from `enumeratedTypeNames[static_cast<int>enumeratedType];`. This is a time/space saving pattern that we should propagate and will hopefully be part of a (near) future pass to maximize the benefits of enumerated types (and compile time constants in general) throughout the code.